### PR TITLE
[ElasticSearch] skipped missing in Shards & sort values are returned in response

### PR DESCRIPTION
**Pull request by daphnesmit**
**Sort Values**
The sort values for each document returned are also returned as part of the response.
See: [sort documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-sort.html#_sort_values)

**ShardsResponse**
Skipped param in ShardsResponse was missing

### CODE REVIEW

**Comment by daphnesmit** [comment]
Sorry forgot to merge upstream!

File: types/elasticsearch/index.d.ts [hunk 1]
```diff
@@ -619,29 +620,30 @@ export interface SearchParams extends GenericParams {
...
-            _version?: number;
-            _explanation?: Explanation;
-            fields?: any;
-            highlight?: any;
-            inner_hits?: any;
-        }>;
-    };
-    aggregations?: any;
-}
+    export interface SearchResponse<T> {
```
Comment (mlamp) [inline types/elasticsearch/index.d.ts]: No tabbing required.

**Review by mlamp — CHANGES_REQUESTED** [review]

**Comment by daphnesmit** [comment]
Fixed tabbing:)

**Review by mlamp — APPROVED** [review]

**Comment by mlamp** [comment]
:+1:

**Pull request merged**